run/archiveloop

```diff
@@ -266,43 +266,42 @@ function disconnect_usb_drives_from_host () {
 }
 
 function archive_teslacam_clips () {
-  log "Checking saved folder count..."
+    log "Checking saved folder count..."
   
-  ensure_cam_file_is_mounted
+    ensure_cam_file_is_mounted
 
-  fix_errors_in_cam_file
+    fix_errors_in_cam_file
   
-  DIR_COUNT=$(find "$CAM_MOUNT"/TeslaCam/SavedClips/* -maxdepth 0 -type d | wc -l)
-  FILE_COUNT=$(find "$CAM_MOUNT"/TeslaCam/SavedClips/* -type f | wc -l)
+    DIR_COUNT=$(find "$CAM_MOUNT"/TeslaCam/SavedClips/* -maxdepth 0 -type d | wc -l)
+    FILE_COUNT=$(find "$CAM_MOUNT"/TeslaCam/SavedClips/* -type f | wc -l)
   
-  log "There are $DIR_COUNT event folder(s) with $FILE_COUNT file(s) to move."
+    log "There are $DIR_COUNT event folder(s) with $FILE_COUNT file(s) to move."
   
-  if [ $DIR_COUNT -gt 0 ]
-  then
-	log "Starting recording archiving..."
-
-	/root/bin/send-push-message "TeslaUSB:" "Archiving $DIR_COUNT event folder(s) with $FILE_COUNT file(s) starting at $(date)"
-
-	if [ -f /root/bin/tesla_api.py ]
-	then
-		while :
-		do
-		sleep 120
-		/root/bin/tesla_api.py wake_up_vehicle &>> "$LOG_FILE"
-		done&
-		WAKE_PID=$!
-		log "wake up car process is $WAKE_PID"
-	fi
-
-	/root/bin/archive-clips.sh
-	
-  fi
-  if [ ! -z "${WAKE_PID+x}" ]
-  then
-    log "killing PID $WAKE_PID"
-    kill $WAKE_PID
-  fi
-  unmount_cam_file
+    if [ $DIR_COUNT -gt 0 ]; then
```

Comment:
please keep "then" on the next line like it was before
(here and elsewhere)

Reply:
No problem, updating incoming